spectree/response.py

```diff
@@ -7,25 +7,31 @@ class Response:
     """
     response object
 
-    :param codes: list of HTTP status code, format('HTTP_[0-9]{3}'), 'HTTP200'
-    :param code_models: dict of <HTTP status code>: <`pydantic.BaseModel`> or None
+    :param args: list of HTTP status code, format('HTTP_[0-9]{3}'), 'HTTP200'
+    :param kwargs: dict of <HTTP status code>: <`pydantic.BaseModel`> or None
+    Note, you can also pass `validate` into the kwargs to disable output validation of this model
     """
 
-    def __init__(self, *codes, **code_models):
-        self.codes = []
-        for code in codes:
-            assert code in DEFAULT_CODE_DESC, 'invalid HTTP status code'
-            self.codes.append(code)
+    def __init__(self, *args, **kwargs):
+
+        self.validate = True
+        self.codes = []  # used to be self.codes
```

Comment:
Not sure if this comment was intentional.

Reply:
woops, no that is left over from an earlier change - let me get rid of them